<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientesForm</class>
 <widget class="QWidget" name="ClientesForm">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelRut">
       <property name="text">
        <string>RUT</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputRut"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelNombre">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="inputNombre"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDireccion">
       <property name="text">
        <string>Dirección</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="inputDireccion"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="inputEmail"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelTelefono">
       <property name="text">
        <string>Teléfono</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="inputTelefono"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelBuscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="inputBuscar"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAgregar">
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditar">
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBuscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tablaClientes">
     <column>
      <property name="text">
       <string>RUT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dirección</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
